roles/download/defaults/main.yml

```diff
@@ -54,6 +54,7 @@ kubeadm_version: "{{ kube_version }}"
 etcd_version: v3.4.3
 
 # gcr and kubernetes image repo define
+gcr_location_default: "eu."
```

Comment:
Add to put this. as `The repository https://quay.io/repository/kubernetes-ingress-controller/nginx-ingress-controller is deprecated and read-only for historical purposes.` and nginx doesn't push to `gcr.io` only to `asia.gcr.io`, `eu.gcr.io` and `us.gcr.io`